config/initializers/cors.rb

```diff
@@ -5,12 +5,13 @@
 
 # Read more: https://github.com/cyu/rack-cors
 
-# Rails.application.config.middleware.insert_before 0, Rack::Cors do
-#   allow do
-#     origins 'example.com'
-#
-#     resource '*',
-#       headers: :any,
-#       methods: [:get, :post, :put, :patch, :delete, :options, :head]
-#   end
-# end
+Rails.application.config.middleware.insert_before 0, Rack::Cors do
+  allow do
+    origins '*'
+
+    resource '*',
+      headers: :any,
+      expose: %w[access-token expiry token-type uid client],
+      methods: [:get, :post, :put, :patch, :delete, :options, :head]
+  end
+end
```

Comment:
You guys already configured CORS in `application.rb`. 
Choose which place you want to configure it. 

(I'd go with this, deleting what you have in application.rb)

Reply:
aha okey, we fix that!